irr/src/COpenGLCoreTexture.h

```diff
@@ -232,7 +232,7 @@ class COpenGLCoreTexture : public ITexture
 #endif
 			GLint max_samples = 0;
 			GL.GetIntegerv(GL_MAX_SAMPLES, &max_samples);
-			MSAA = std::min(MSAA, (u8)max_samples);
+			MSAA = core::min_(MSAA, (u8)max_samples);
```

Comment:
Why this in this PR? I see no difference in no SDL build.

Reply:
MSVC build started failing after disabling sdl...